### PR TITLE
OPENEUROPA-1322: Use the proper Docker way of overriding docker-composer.yml file.

### PR DESCRIPTION
**Pull request by drupol**
OPENEUROPA-1322: Use the proper Docker way of overriding docker-composer.yml file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,37 +171,45 @@ services to be up and running.
...
-$ ./vendor/bin/run sparql:reset
-```
+
+## Contributing
+
+Please read [the full documentation](https://github.com/openeuropa/openeuropa) for details on our code of conduct, and the process for submitting pull requests to us.
+
+## Versioning
+
+We use [SemVer](http://semver.org/) for versioning. For the available versions, see the [tags on this repository](https://github.com/openeuropa/drupal_module_template/tags).
```
Comment (dxvargas) [inline README.md]: Wrong link, it must refer to oe_content .

File: composer.json [hunk 2]
```diff
@@ -7,15 +7,14 @@
     "prefer-stable": true,
     "require": {
         "php": "^7.1",
-        "drupal/core": "~8.6@alpha",
+        "openeuropa/drupal-core-require": "^8.6",
```
Comment (dxvargas) [inline composer.json]: Here version is good but then "openeuropa/drupal-core-require-dev" should also be "^8.6".
It should appear in alphabetical order.

File: README.md (hunk 1 above)
Comment (drupol) [reply README.md]: Fixed.

File: composer.json (hunk 2 above)
Comment (drupol) [reply composer.json]: Fixed.

File: README.md [hunk 3]
```diff
@@ -63,74 +72,96 @@ You can build the test site by running the following steps.
...
 ```
 
 * Customize build settings by copying `runner.yml.dist` to `runner.yml` and
 changing relevant values, like your database credentials.
 
 * Setup test site by running:
 
-```
-$ ./vendor/bin/run drupal:site-setup
+```bash
```
[the remark this answers is not in the document]
Comment (drupol) [reply README.md]: Fixed.

**Pull request merged**